fc/jira/issue.py

```diff
@@ -155,3 +155,10 @@ def comment(self, note: str):
         response = requests.post('{}{}/comment'.format(self.api_url, self.id), json=json,
                                  auth=HTTPBasicAuth(self.auth.username(), self.auth.password()))
         response.raise_for_status()
+
+    def watch(self, username: str):
+        data = "\"{}\"".format(username)
```

Comment:
@halprin: Any preference here on how the json data is created?  Whether as a `string` like this, or as an actual json object?  I think we treat it as a json object everywhere else.

Reply:
```suggestion
        data = '"{}"'.format(username)
```
Or even changing the variable name for extra credit.
```suggestion
        request_body = '"{}"'.format(username)
```
Yeah, @sfradkin, one could do...
```suggestion
        request_body = json.dumps(username)
```
which results in the same thing.

Reply:
OK, so I thought it was "not really JSON" but that is because [the meaning of this post whooshed over my head late in the afternoon](https://community.atlassian.com/t5/Answers-Developer-Questions/Please-Help-Got-error-400-when-trying-to-add-watcher-using-REST/qaq-p/567259). Now it makes sense `"property"` is valid JSON after all?

Reply:
well nevermind because looking closer at the actual API, it deviates from other JIRA APIs and doesn't actually require valid JSON as the request body.  In fact, it apparently doesn't even allow you to add more than one watcher at a time.

https://jira.atlassian.com/browse/JRASERVER-29304